docs/getting-started-guides/ubuntu/local.md

```diff
@@ -13,48 +13,23 @@ The purpose of using [LXD](https://linuxcontainers.org/lxd/) on a local machine
 {% capture prerequisites %}
 ## Prerequisites
```

Comment:
Sorry, I forgot to mention that it automatically adds a header for this section titled *Before you begin*, so you can remove this one.